meta/main.yml

```diff
@@ -7,4 +7,4 @@ galaxy_info:
 
 dependencies:
   - src: git+https://github.com/appuio/ansible-module-openshift.git
-    version: v1.3.4
+    version: v1.5.0
```

Comment:
Even surprised, that this didn't crash before. Usage of the k8s module would be better, but I think this still works good enough.

Reply:
ansible-galaxy complains about roles having mismatched versions of dependencies and picks one, I didn't check which one.